app/javascript/components/Articles/ArticlesList.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import ArticlesDetails from './ArticlesDetails';
+import { array } from "prop-types";
+
+export interface ArticlesListProps {
```

Comment:
you have created an interface here but I don't see it used?

Reply:
Good catch! I have updated the usage issue in the most recent commit, though I'm unsure of what type to assign the articles to... The types of `Object` or `JSON` do not fully encompass the uses this data serves and causes webpack errors. 

For the moment, I have given them the type of `any`, which while a small bit of a cop out, works well and doesn't increase the complexity of the component. 

Any feedback would be appreciated.

Reply:
SO what you could do is create another interface for it. You could then define it as an array of that interface. See SeamlessAccount in portal app/webpack/components/seamless_login_access/SeamlessLoginAccess.tsx. Its used as an array in this file but is itself an interface. Hope this makes sense

Reply:
This is a great point of reference that I will definitely investigate! I think the current implementation fits our purposes for the moment though.